app/forms/admin/school_transfer_form.rb

```diff
@@ -76,17 +76,32 @@ def perform_transfer!
   end
 
   def cannot_transfer_to_new_school?
-    cannot_transfer_reason.present?
+    no_programmes_to_transfer_into_or_continue?
   end
 
   def cannot_transfer_reason
-    if new_school_cohort.blank?
-      "they have not chosen a programme for #{participant_cohort.start_year}"
-    elsif new_school_cohort.induction_programmes.full_induction_programme.empty? && new_school_cohort.induction_programmes.core_induction_programmes.empty?
-      "they do not have any FIP or CIP induction programmes for #{participant_cohort.start_year}"
+    if no_programmes_to_transfer_into_or_continue?
+      "they have no programmes available for #{participant_cohort.start_year} and there is no existing programme to continue"
     end
   end
 
+  def skip_transfer_options?
+    # are there any options to select for the transfer
+    programme_count = new_school_cohort&.induction_programmes&.count || 0
```

Comment:
Could this bit be shortened a bit by assigning `new_school_cohort.induction_programmes.first` to a variable and using it below? As the logic is quite complex it would benefit from its own specs too.